libs/text-splitters/langchain_text_splitters/base.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         length_function: Callable[[str], int] = len,
         keep_separator: bool = False,
         add_start_index: bool = False,
+        add_chunk_id: bool = False,
```

Comment:
Named arguments should be introduced at end of list to avoid breaking changes (some users may be invoking by position). If you place it after `strip_whitespace` it'll be an additive change.

I'll need a bit more time to review the change here, but let's see if we can get the code to be vectorstore agnostic?

Reply:
Got it. Will change the ordering

Reply:
Updated this.